frontend/app/src/premium/register-components.ts

```diff
@@ -129,6 +130,8 @@ export function registerComponents() {
   Vue.component('GraphTooltipWrapper', GraphTooltipWrapper);
   // Version 19 - 1.26
   Vue.component('LpPoolIcon', LpPoolIcon);
+  // Version 20 - 1.27
+  Vue.component('BadgeDisplay', BadgeDisplay);
```

Comment:
I will take care of it, but in order for this to properly work and not try to load we also have to do a bump here:

https://github.com/rotki/rotki/blob/a414eec6bbd51430dc01fd982b2805f7af86325e/frontend/app/src/premium/setup-interface.ts#L101